hpedockerplugin/hpe/hpe_3par_mediator.py

```diff
@@ -1033,6 +1033,51 @@ def create_vfs(self, vfs_name, ip, subnet, cpg=None, fpg=None,
         finally:
             self._wsapi_logout()
 
+    def set_ACL(fUName, fGName, fMode):
+        LOG.info("Inside set ACL call but temperory not making"
+                 " any rest call.")
+
+    def _check_usr_grp_existence(self, fUserOwner, res_cmd):
+        fuserowner = str(fUserOwner)
+        uname_index = 0
+        uid_index = 1
+        user_name = None
+        first_line = res_cmd[1]
+        first_line_list = first_line.split(',')
+        for index, value in enumerate(first_line_list):
+            if value == 'Username':
+                uname_index = index
+            if value == 'UID':
+                uid_index = index
+        res_len = len(res_cmd)
+        end_index = res_len - 3
+        for line in res_cmd[2:end_index]:
+            line_list = line.split(',')
+            if fuserowner == line_list[uid_index]:
+                user_name = line_list[uname_index]
+                return user_name
+        if user_name is None:
+            return None
+
+    def usr_check(self, fUser, fGroup):
+        LOG.info("I am inside usr_check")
+        cmd1 = ['showfsuser']
+        cmd2 = ['showfsgroup']
+        try:
+            LOG.info("Now will execute first cmd1")
+            cmd1.append('\r')
```

Comment:
This can be done above at line #1064 itself as
cmd1 = ['showfsuser', '\r']

Reply:
OKay we can take this in upcoming PR